gtk2_ardour/wscript

```diff
@@ -622,7 +620,8 @@ def build(bld):
         obj.source    = list(gtk2_ardour_sources)
         obj.target = 'ardour-' + str (bld.env['VERSION'])
         obj.includes = ['.']
-        obj.ldflags = ['-no-undefined']
+        obj.includes += bld.env['INCLUDES_LILV']
+        obj.ldflags = ['-l' + lib for lib in bld.env['LIB_LILV']]
```

Comment:
Does `obj.uselib += ['LILV']` help instead of this?

Reply:
I'll check and update

Reply:
Seems this didn't work, because:

`AttributeError: 'task_gen' object has no attribute 'uselib'`